api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md

```diff
@@ -8,4 +8,124 @@ default: undefined
 Specifies a series color.
 
 ---
-Use this property to paint all pie slices in one color.
\ No newline at end of file
+You can use this property to paint all pie slices in one color.
+
+#include common-colorlist
+
+You can also specify a custom pattern or gradient instead of a plain color. Implement the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) or [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to get an id. Then, do the following:
+
+1. Specify the `base` color for labels and connectors in the **color** configuration object.
+
+1. Set the `fillId` field to the obtained id.
```

Comment:
I would maybe move this item to second position, to keep related items together (they both mention fill ID).

```suggestion
1. Set the `fillId` field to the obtained ID.
```

Reply:
Noted